app/controllers/calls_controller.rb

```diff
@@ -7,7 +7,7 @@ class CallsController < ApplicationController
   # CallSid - default param from twilio (required)
   def start
     response = Twilio::TwiML::Response.new do |r|
-      r.Play AudioFileFetcher.file_for_key('intro_message')
+      r.Play AudioFileFetcher.audio_url_for_key('intro_message')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.